src/components/Block/Strategy.vue

```diff
@@ -1,35 +1,27 @@
 <template>
-  <Block>
-    <div class="d-flex flex-items-baseline">
-      <h3>
-        <a
-          v-text="strategy.key"
-          :href="
-            `https://github.com/snapshot-labs/snapshot.js/tree/master/src/strategies/${strategy.key}`
-          "
-          target="_blank"
-        />
-      </h3>
-      <div class="ml-1">v{{ strategy.version }}</div>
-    </div>
-    <div>
+  <router-link :to="fromModal ? {} : `/strategy/${strategy.key}`">
```

Comment:
There is already a link in parent component for modal. I think the link should not be part of the component here https://github.com/balancer-labs/snapshot/pull/190/files#diff-1f2854b767931992cb7464d658f2eec7525e766df4e1377dc0079d0cd6f8dff4R28-R33
But be in the explore page instead here https://github.com/balancer-labs/snapshot/blob/e15cd432ad436a9ffcb308ea10a9e345fd0e1b24/src/views/Explore.vue#L26